includes/model/class-application-user.php

```diff
@@ -52,7 +50,7 @@ public function get_name() {
 	}
 
 	public function get_preferred_username() {
-		return $this::get_name();
+		return self::get_name();
```

Comment:
this is not a static function
```suggestion
		return $this->get_name();
```